data-generation/src/main/scala/SnapshotDelta.scala

```diff
@@ -61,13 +62,13 @@ object SnapshotDeltaObject {
   def create[VD,ED](logs: RDD[LogTSV], snapType: SnapshotIntervalType): SnapshotDelta[Attributes, Attributes] = {
     snapType match {
       case SnapshotIntervalType.Time(duration) => snapShotTime(logs, duration)
-      case SnapshotIntervalType.Count(numberOfActions) => snapShotCount(logs, numberOfActions)
+      case SnapshotIntervalType.Count(numberOfActions) => createSnapShotCountModel(logs, numberOfActions)
     }
   }
 
-  def snapShotCount(logs: RDD[LogTSV], numberOfActions: Int): SnapshotDelta[Attributes, Attributes] = {
+  def createSnapShotCountModel(logs: RDD[LogTSV], numberOfActions: Int): SnapshotDelta[Attributes, Attributes] = {
```

Comment:
```suggestion
  def createSnapshotCountModel(logs: RDD[LogTSV], numberOfActions: Int): SnapshotDelta[Attributes, Attributes] = {
```
SnapShot -> Snapshot